lessons/Lesson 1a.ipynb

```diff
@@ -440,21 +530,21 @@
  ],
  "metadata": {
```

Comment:
Opening the notebook in python3 and saving it updated this metadata automatically.